server/app.js

```diff
@@ -83,18 +98,23 @@ async function main() {
     ));
 
     app.use('/', (req, res, next) => {
-      req.db = db;
+      req.db = client;
```

Comment:
We would probably not need to pass client on the Request object for each router anymore right?

Reply:
Fixed